models/quality_measures/intermediate/nqf0053_osteoporosis_management_in_women_who_had_a_fracture/quality_measures__int_nqf0053_numerator.sql

```diff
@@ -0,0 +1,270 @@
+        , denominator.measure_id
+        , denominator.measure_name
+        , denominator.measure_version
+        , denominator.recorded_date
+        , denominator.performance_period_begin
+        , denominator.performance_period_end
+    from procedures_osteo_related
+    inner join denominator
+        on procedures_osteo_related.patient_id = denominator.patient_id
+        and 
+            procedures_osteo_related.procedure_date between
+            denominator.recorded_date 
+            and
+            {{dbt.dateadd (
+                      datepart = "month"
+                    , interval = +6
+                    , from_date_or_timestamp = "denominator.recorded_date"
+                          )
+            }} 
+
+)
+
+, denominator_patients_disqualified_from_procedure as (
+    
+    select 
+          denominator.patient_id
+        , denominator.measure_id
+        , denominator.measure_name
+        , denominator.measure_version
+        , denominator.recorded_date
+        , denominator.performance_period_begin
+        , denominator.performance_period_end  
+    from denominator
+    left join qualifying_procedures 
+    on denominator.patient_id = qualifying_procedures.patient_id
+    where qualifying_procedures.patient_id is null
+
+)
+
+-- pharmacy_claim begin
+
+, osteo_rx_codes as (
+
+    select
+          code
+        , code_system
+        , concept_name
+    from value_sets
+    where lower(concept_name) 
+        in 
+        ( 
+          'osteoporosis medications for urology care'
+        , 'osteoporosis medication'
+        , 'bisphosphonates'
+        )
+
+)
+
+, pharmacy_claims_osteo_related as (
+
+    select
+        patient_id
+      , dispensing_date
+      , ndc_code  
+    from {{ref('quality_measures__stg_pharmacy_claim')}} as pharmacy_claims
+    inner join osteo_rx_codes
+    on pharmacy_claims.ndc_code = osteo_rx_codes.code
+        and lower(osteo_rx_codes.code_system) = 'ndc'
+
+)
+
+, qualifying_pharmacy_claims as (
+
+    select pharmacy_claims_osteo_related.*
+        , denominator_patients_disqualified_from_procedure.measure_id
+        , denominator_patients_disqualified_from_procedure.measure_name
+        , denominator_patients_disqualified_from_procedure.measure_version
+        , denominator_patients_disqualified_from_procedure.recorded_date
+        , denominator_patients_disqualified_from_procedure.performance_period_begin
+        , denominator_patients_disqualified_from_procedure.performance_period_end
+    from pharmacy_claims_osteo_related
+    inner join denominator_patients_disqualified_from_procedure
+        on pharmacy_claims_osteo_related.patient_id = denominator_patients_disqualified_from_procedure.patient_id
+        and pharmacy_claims_osteo_related.dispensing_date 
+            between 
+            denominator_patients_disqualified_from_procedure.performance_period_begin and denominator_patients_disqualified_from_procedure.recorded_date
```

Comment:
I think this date range is wrong. Shouldn't this be the same as the qualifying_medications CTE? The p[harmacy claim dispensed date should be between the fracture date and 6 months after the fracture.

Reply:
It has been updated as in qualifying_medications CTE. I had perceived it as medications already taken before the date of fracture within performance period. Thank you for confirming.